### ISSUE TITLE
"Could not guess mimetype" warnings should be suppressed in appengine app.yaml 

### ISSUE THREAD

**Issue by danbri** [schemaorg/schemaorg#1340]
Every time we upload it complains about numerous files with .nt, .jsonld, .nq suffixes from data/releases/*. We fixed this warning for the single .owl file but need to do same for regex-defined patterns.

I experimented unsuccessfully along these lines:

```
# Attempt to avoid many warnings of this form:
#   "Could not guess mimetype for data/releases/3.1/all-layers.jsonld.  Using application/octet-stream."
#   "Could not guess mimetype for data/releases/3.0/ext-auto.nt.  Using application/octet-stream."
# using *
- url: /data/releases/*/*.nt # URL path seems required here, we didn't expose /data/ previously.
  static_files: data/releases/*/(*.nt)
  upload: data/releases/*/\1
  mime_type: application/n-triples
```

This particular snippet is broken, subtleties are around which '.' needs \ escaping, and handling the larger set of paths. I started here: http://stackoverflow.com/questions/20425364/could-not-guess-mimetype
